app/javascript/lib/metrics-api.js

```diff
@@ -339,14 +356,19 @@ function getAnalyticsPageName() {
  * @param {String} name
  * @param {Object} props
  */
-export function log(name, props={}) {
+export function log(name, props = {}) {
```

Comment:
I hadn't realized [JavaScript conventionally puts a space around "=" in default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)!  That differs from Python and Ruby convention, unfortunately, but I agree we should use this style in JS.